idlpy/src/types.h

```diff
@@ -17,4 +17,8 @@
 
 idl_retcode_t generate_types(const idl_pstate_t *pstate, idlpy_ctx ctx);
 
+//Reserved Python keywords support (Issue 105)
+const char *get_c_name(const char *name);
```

Comment:
Maybe this function should live in `naming.h` instead?

Reply:
Ok. That header seems more proper to contain the method. We will move it.